devdocs/settingUp.md

```diff
@@ -77,7 +77,8 @@ Important: When a version is specified, please install that version instead of t
    e.g. Change <b>`http://teammates-john.appspot.com`</b>`/page/instructorCourseJoin?key=F2AD69F8994BA92C8D605BAEDB35949A41E71A573721C8D60521776714DE0BF8B0860F12DD19C6B955F735D8FBD0D289&instructorinstitution=NUS` <br>
    to <b>`http://localhost:8888`</b>`/page/instructorCourseJoin?key=F2AD69F8994BA92C8D605BAEDB35949A41E71A573721C8D60521776714DE0BF8B0860F12DD19C6B955F735D8FBD0D289&instructorinstitution=NUS`
 8. Now, to access the dev server as a student, first make sure you are logged in as an instructor. Add a course for yourself and then add the students for the course.<br>
-   After that, log in as admin and search for the student you added in as instructor. From the search results, use the join link adjacent to the student you added.<br>
+   After that, log in as admin by going to `http://localhost:8888/admin/adminSearchPage` and provide the same GoogleID you used for logging in step 6.<br>
+   Search for the student you added in as instructor. From the search results, click anywhere on the desired row(except on the student name) to get the Course Join Link for that student.<br>
```

Comment:
The first letters in "Course Join Link" doesn't need to be capitalised. "course join link" should be fine, since "join link" appears elsewhere in the instructions.